### ISSUE TITLE
Unable to Add/Edit Alt Text to images when editing posts

### ISSUE THREAD

**Issue by rxbrad** [trunks-social/trunks-public#278]
When editing a post, there is no option to add/edit the Content Description on existing images (or on images that you just added via an edit). You can only delete images. 

![Screenshot_20231029-132620](https://github.com/trunks-social/trunks-public/assets/13750413/b4c99dca-9df6-40af-8bd0-a74d9fca4e3e)

Android v1.2.9:174
